src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -194,10 +223,7 @@ void PerformAudioPassThruRequest::SendSpeakRequest() {
   using namespace smart_objects;
 
   SmartObject msg_params = smart_objects::SmartObject(SmartType_Map);
-
-  if ((*message_)[str::msg_params].keyExists(str::initial_prompt) &&
-      (0 < (*message_)[str::msg_params][str::initial_prompt].length())) {
```

Comment:
@VProdanov it is ok because we call method "SendSpeakRequest"  when the same check returns true please see line 101.